tests/stochastic/generate_test_h2o_network.py

```diff
@@ -0,0 +1,276 @@
+import os

```

Comment:
I feel like this shouldn't be directly in tests. There should be, like, a tests/scripts or tests/generate subdirectory for all of the files we use to generate test files.

Reply:
Sure. Easy enough to change this.